content/influxdb/cloud-dedicated/get-started/write.md

```diff
@@ -704,7 +705,163 @@ To write data to {{% cloud-name %}} using Node.js, use the
 {{% /influxdb/custom-timestamps %}}
 <!------------------------------- END NODE.JS CONTENT ------------------------------>
 {{% /tab-content %}}
+{{% tab-content %}}
+<!---------------------------- BEGIN C# CONTENT --------------------------->
+1.  If you haven't already, follow the [Microsoft.com download instructions](https://dotnet.microsoft.com/en-us/download) to install .NET and the `dotnet` CLI.
+2. In your terminal, create an executable C# project using the .NET **console** template.
+
+    ```sh
+    dotnet new console --name influxdb_csharp_client
+    ```
+
+3. Change into the generated `influxdb_csharp_client` directory.
+    
+    ```sh
+    cd influxdb_csharp_client
+    ```
+
+4. Run the following command to install the latest version of the InfluxDB v3 C# client library.
+
+    ```sh
+    dotnet add package InfluxDB3.Client
+    ```
+
+5.  In your editor, create a `Write.cs` file and enter the following sample code:
+
+    ```c#
```

Comment:
Does this codeblock need to be wrapped in the `{{% influxdb/custom-timestamps %}}` shortcode or are all the tabs already wrapped in it?

Reply:
Yep, I missed it. Thanks!